Sources/SwiftDocC/Infrastructure/NodeURLGenerator.swift

```diff
@@ -54,6 +54,8 @@ public struct NodeURLGenerator {
         case documentation(path: String)
         case documentationCuration(parentPath: String, articleName: String)
         case article(bundleName: String, articleName: String)
+        case tutorialTableOfContents(name: String)
+        @available(*, deprecated, renamed: "tutorialTableOfContents(name:)", message: "Use 'tutorialTableOfContents(name:)' instead. This deprecated API will be removed after 6.1 is released")
```

Comment:
This should be `6.2`:
```suggestion
        @available(*, deprecated, renamed: "tutorialTableOfContents(name:)", message: "Use 'tutorialTableOfContents(name:)' instead. This deprecated API will be removed after 6.2 is released")
```

Reply:
Updated in https://github.com/swiftlang/swift-docc/pull/1039/commits/5b43b7c13da335751b0ff3c024db7674ec8dc7b9